docs/Server.md

```diff
@@ -352,7 +352,8 @@ You can change this default setting by passing the option `querystringParser` an
 ```js
 const qs = require('qs')
 const fastify = require('fastify')({
-  querystringParser: str => qs.parse(str)
+  // if caseSenstive is set to false, then parse str in lowercase
+  querystringParser: str => qs.parse(!fastify.initConfig.caseSensitive ? str.toLowerCase() : str)
```

Comment:
The `caseSensitive` option has nothing to do with query parameters -- https://github.com/fastify/fastify/blob/36bca513cbcb17f0797ef69ead4e6c7afa831b98/docs/Server.md#casesensitive

Again, the proposed resolution in the issue is to clarify that the `qs` module should be used if case insensitivity is desired for query parameters.